### PR TITLE
Update Copyright information

### PR DESCRIPTION
**Pull request by nitesh3108**
# Description
Update Copyright information.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/491|

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Successfully Build podman image.

### CODE REVIEW

**Review by prablr79 — APPROVED** [review]
lgtm

**Review by HarishH-DELL — APPROVED** [review]

File: csi-utils/csiutils.go [hunk 1]
```diff
@@ -1,5 +1,21 @@
 package csiutils
 
+/*
+Copyright (c) 2021 Dell Inc, or its subsidiaries.
```
Comment (rajkumar-palani) [inline csi-utils/csiutils.go:4]: Copyright (c) 2021 - 2022 Dell Inc, or its subsidiaries.

Comment (rajkumar-palani) [reply csi-utils/csiutils.go:4]: Approving it, considering no changes done in 2022.

**Review by rajkumar-palani — COMMENTED** [review]
lgtm

**Pull request merged**